resources/css/bem/user-search-card.less

```diff
@@ -71,10 +72,16 @@
 
     &--username {
       .link-plain();
+
+      display: inline-block;
```

Comment:
it ended up just being `block` because flex container can only contain block layout elements